webview/src/plots/components/customPlots/customPlotsSlice.ts

```diff
@@ -43,13 +44,14 @@ export const customPlotsSlice = createSlice({
       if (!action.payload) {
         return customPlotsInitialState
       }
-      const { plots, ...statePayload } = action.payload
+      const { plots, colors, ...statePayload } = action.payload
       const plotsIds = plots?.map(plot => plot.id) || []
       const snapShots = addPlotsWithSnapshots(plots, Section.CUSTOM_PLOTS)
       removePlots(plotsIds, Section.CUSTOM_PLOTS)
       return {
         ...state,
         ...statePayload,
+        colors: colors || { domain: [], range: [] },
```

Comment:
`{ domain: [], range: [] }` could be a const. `const initialColorsState = { domain: [], range: [] }`.